### ISSUE TITLE
Acceptance test

### ISSUE THREAD

**Issue by MadsBloendAndersen** [Energinet-DataHub/green-energy-hub#488]

**Comment by energinet-ajw** [comment]
Notes. 
- The A.C. tests are implemented as domain tests on all features. There might be exceptions.
- These tests might change when the test repport is released.

Questions
- What kind of overview?
- Why must the list of tests be documented on this issue? Would it be better to generate a list of the actual tests, written in plain English so SME can read it?
- Should we tag AC tests?
